### PR TITLE
AssetTree theme fix

### LINKED ISSUE #311: Theme support in AssetTree
Add Theme support in AssetTree component.

### PR DESCRIPTION
**Pull request by joyrostie**
AssetTree theme fix

Closes #311

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-319.gearboxjs.preview.cogniteapp.com

File: src/components/ThemeProvider/stories/ThemeProvider.stories.tsx [hunk 1]
```diff
@@ -93,7 +93,7 @@ storiesOf('ThemeProvider/Examples', module)
       return (
```
[the remark this answers is not in the document]
Comment (joyrostie) [reply src/components/ThemeProvider/stories/ThemeProvider.stories.tsx]: Interesting notice :-) I will create corresponding readme now.

**Comment by joyrostie** [comment]
> Pretty done.
> What about styling of selected element (with custom color for example)?

The problem is that to color the displayed text we need to go deeper to span element than resides inside of li element generated by antd TreeNode node and owns the text.
Using theming it's possible to configure the css selector in styled-component and pass the value it to achieve our goal but using styling way that applies styles inline to react node it is impossible to pass css selector itself.

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-319.gearboxjs.preview.cogniteapp.com

**Comment by mrtraser** [comment]
but you can redefine antd class name css value inside styled component, like: 
```css
&.ant-tree-node-selected {
    background-color: black!important;
}
```

**Comment by joyrostie** [comment]
> but you can redefine antd class name css value inside styled component, like:
> 
> ```css
> &.ant-tree-node-selected {
>     background-color: black!important;
> }
> ```

Added.

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-319.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-319.gearboxjs.preview.cogniteapp.com

**Pull request merged**